integration/ide/gen-idea/resources/hello-idea/idea/modules.xml

```diff
@@ -16,6 +16,8 @@
             <module fileurl="file://$PROJECT_DIR$/.idea/mill_modules/moduled.iml" filepath="$PROJECT_DIR$/.idea/mill_modules/moduled.iml"/>
             <module fileurl="file://$PROJECT_DIR$/.idea/mill_modules/modulee.iml" filepath="$PROJECT_DIR$/.idea/mill_modules/modulee.iml"/>
             <module fileurl="file://$PROJECT_DIR$/.idea/mill_modules/modulef.iml" filepath="$PROJECT_DIR$/.idea/mill_modules/modulef.iml"/>
+            <module fileurl="file://$PROJECT_DIR$/.idea/mill_modules/moduleg-2.12.5.iml" filepath="$PROJECT_DIR$/.idea/mill_modules/moduleg-2.12.5.iml"/>
```

Comment:
The fix changes this and the file name itself from `moduleg.2.12.5.iml` to `moduleg-2.12.5.iml`